.fasterci/config.yaml

```diff
@@ -16,3 +16,31 @@ workflows:
                     - //...
                 test_targets:
                     - //...
+          - name: e2e/link_js_package
```

Comment:
hey, thanks - could we use the same matrix syntax as in the github actions configuration?

Reply:
this is what I'm referring to: https://github.com/aspect-build/rules_js/blob/main/.github/workflows/ci.yaml#L19-L25

Reply:
Using yaml refs is recommended way to achieve this. see https://github.com/adobe/rules_gitops/blob/fd76291f513fce4fc637c22f8055b3a49ad82fe7/.fasterci/config.yaml#L32 for example.